scripts/run_ansible_test

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+## Shell Opts ----------------------------------------------------------------
+
+set -o pipefail
+set -xeuo
+
+## Vars ----------------------------------------------------------------------
+PROJECT_DIR="$(dirname $(readlink -f ${BASH_SOURCE[0]}))/../"
+USE_VENV=${USE_VENV:-yes}
+HOME=${HOME:-/tmp}
+
+ansible_test='ansible-test'
+collection_path='/usr/share/ansible/collections'
+case ${USE_VENV} in
+    y|yes|true):
+        ansible_test="${HOME}/test-python/bin/ansible-test"
+        collection_path="${HOME}.ansible/collections/ansible_collections"
+        ;;
+esac
+
+mkdir -p ${HOME}/code/ansible_collections/cifmw/general
+cp -ra ${PROJECT_DIR}ci_framework/* ${HOME}/code/ansible_collections/cifmw/general/
+ln -s ${collection_path}/* ${HOME}/code/ansible_collections/
```

Comment:
In case we run local we need to manually cleanup the `ansible_collections` folder otherwise will give error. Not a problem if it will be used just in CI.

Reply:
imho we should run in container using the new `run_ctx_ansible_test` target - that will start a container with the `--rm` flag in order to get a env as clean as possible for each run.